components/funder/src/handler/handle_control.rs

```diff
@@ -620,7 +620,7 @@ where
     ));
     m_state.mutate(funder_mutation);
 
-    // Remove ourselves from the remaining route.
+    // Remove the next node from the remaining route.
```

Comment:
@pzmarzly : Actually here your original comment was correct. We remove ourselves from the remaining route. See [Line 542](https://github.com/freedomlayer/offst/blob/d206f9dd07cd3c6e0e7286bbf37f1c7a0f5ee57f/components/funder/src/handler/handle_control.rs#L542).